### DISCUSSION TITLE
Maintain cache while PM4K is open and not destroying cache when leaving library view.

### DISCUSSION THREAD

**Discussion by Uncloak2** [pannal/plex-for-kodi#149] [Ideas]
title

**Comment by pannal** [comment]
There is no cache to maintain. The poster caching is handled by Kodi.

**Comment by pannal** [comment]
Do you want a data cache?

**Reply by Uncloak2** [reply]
> Do you want a data cache?

If it means items are remembered when exiting library views and re-entering, sure that would be great.

**Comment by pannal** [comment]
This is implemented in the current 0.8.0 beta!
